packages/contracts-bedrock/test/DeploySuperchain.t.sol

```diff
@@ -204,75 +180,77 @@ contract DeploySuperchainOutput_Test is Test {
-        roles: DeploySuperchainInput.Roles({
-            proxyAdminOwner: makeAddr("defaultProxyAdminOwner"),
-            protocolVersionsOwner: makeAddr("defaultProtocolVersionsOwner"),
-            guardian: makeAddr("defaultGuardian")
-        }),
-        paused: false,
-        requiredProtocolVersion: ProtocolVersion.wrap(1),
-        recommendedProtocolVersion: ProtocolVersion.wrap(2)
-    });
+    // Define default input variables for testing.
+    address defaultProxyAdminOwner = makeAddr("defaultProxyAdminOwner");
+    address defaultProtocolVersionsOwner = makeAddr("defaultProtocolVersionsOwner");
+    address defaultGuardian = makeAddr("defaultGuardian");
+    bool defaultPaused = false;
+    ProtocolVersion defaultRequiredProtocolVersion = ProtocolVersion.wrap(1);
+    ProtocolVersion defaultRecommendedProtocolVersion = ProtocolVersion.wrap(2);
 
     function setUp() public {
         deploySuperchain = new DeploySuperchain();
-        (dsi, dso) = deploySuperchain.getIOContracts();
+        (dsi, dso) = deploySuperchain.etchIOContracts();
     }
 
     function unwrap(ProtocolVersion _pv) internal pure returns (uint256) {
         return ProtocolVersion.unwrap(_pv);
     }
 
-    function test_run_memory_succeeds(DeploySuperchainInput.Input memory _input) public {
-        vm.assume(_input.roles.proxyAdminOwner != address(0));
-        vm.assume(_input.roles.protocolVersionsOwner != address(0));
-        vm.assume(_input.roles.guardian != address(0));
-
-        DeploySuperchainOutput.Output memory output = deploySuperchain.run(_input);
-
-        // Assert that individual input fields were properly set based on the input struct.
-        assertEq(_input.roles.proxyAdminOwner, dsi.proxyAdminOwner(), "100");
-        assertEq(_input.roles.protocolVersionsOwner, dsi.protocolVersionsOwner(), "200");
-        assertEq(_input.roles.guardian, dsi.guardian(), "300");
-        assertEq(_input.paused, dsi.paused(), "400");
-        assertEq(unwrap(_input.requiredProtocolVersion), unwrap(dsi.requiredProtocolVersion()), "500");
-        assertEq(unwrap(_input.recommendedProtocolVersion), unwrap(dsi.recommendedProtocolVersion()), "600");
+    function hash(bytes32 _seed, uint256 _i) internal pure returns (bytes32) {
+        return keccak256(abi.encode(_seed, _i));
+    }
 
-        // Assert that individual output fields were properly set based on the output struct.
-        assertEq(address(output.superchainProxyAdmin), address(dso.superchainProxyAdmin()), "700");
-        assertEq(address(output.superchainConfigImpl), address(dso.superchainConfigImpl()), "800");
-        assertEq(address(output.superchainConfigProxy), address(dso.superchainConfigProxy()), "900");
-        assertEq(address(output.protocolVersionsImpl), address(dso.protocolVersionsImpl()), "1000");
-        assertEq(address(output.protocolVersionsProxy), address(dso.protocolVersionsProxy()), "1100");
+    function testFoo() public {
```

Comment:
Rename? Where is the seed taken from?

Reply:
Oops, this was debugging a failed CI run. Will delete

Reply:
Deleted in bd36605f4